From dc55f7b094f88f3b2c98367cb771be7aa31120a1 Mon Sep 17 00:00:00 2001
From: Serhii Kupriienko
Date: Tue, 4 Feb 2025 16:43:15 +0200
Subject: [PATCH] disable the check-libstd++.py test

---
 src/meson.build | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/meson.build b/src/meson.build
index b9daabf01..403dfffb0 100644
--- a/src/meson.build
+++ b/src/meson.build
@@ -1033,7 +1033,7 @@ if get_option('tests').enabled()
   if cpp.get_argument_syntax() != 'msvc' and not meson.is_cross_build() # ensure the local tools are usable
     dist_check_script += ['check-static-inits', 'check-symbols']
     if get_option('wasm').disabled() and not get_option('with_libstdcxx')
-      dist_check_script += ['check-libstdc++']
+      #dist_check_script += ['check-libstdc++']
     endif
   endif
 
-- 
2.48.0

